src/main/java/com/google/api/codegen/config/DiscoApiModel.java

```diff
@@ -62,6 +62,12 @@ public boolean hasMultipleServices() {
     return interfaceModels;
   }
 
+  @Override
+  public Iterable<? extends TypeModel> getTypes() {
+    // TODO: is this supported?
```

Comment:
@landrito @andreamlin I believe this is supported in discovery (i.e., getting the types listed in the `types` section of the Service config); could one of you confirm?

Reply:
(This doesn't need to be addressed in this PR, but I'd prefer a filed issue to a naked `TODO`.)

Reply:
The types listed in the types section of the service config will be mapped to a schema in the schemas list inside the discovery document. I am unsure how that maps to disco-gen though.